examples/agora-vote/hardhat.config.cjs

```diff
@@ -0,0 +1,15 @@
+require('dotenv').config();
+const { optimism } = require('viem/chains');
```

Comment:
Is there any issue with using public rpc for this? lol

Reply:
Only issue I can see is potentially blocking CI but not a huge deal 🤷